python/setup.py

```diff
@@ -378,12 +379,14 @@ def api_main(program, *args):
     result = None
 
     if parsed_args.command == "build":
-        kwargs = dict(build_python=False, build_java=False)
+        kwargs = dict(build_python=False, build_java=False, build_cpp=False)
```

Comment:
Change `default="python"` to `default="python,cpp"`.

Reply:
Good!